taxi/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from .views import index
```

Comment:
Use absolute imports instead of the relative ones here and in the other places in the future